### ISSUE TITLE
feature request: image descriptions in media blocks

### ISSUE THREAD

**Issue by jerwarren** [valknight/Cohost.py#7]
Would love to be able to programmatically attach descriptions to images

**Comment by valknight** [comment]
Oh huh! I didn't even realise cohost had alt text support now. I'll totally get on this ASAP :)

Thanks for the heads up!

**Comment by valknight** [comment]
(marking as a bug as I want to treat missing accessibility stuff as such)

**Comment by valknight** [comment]
done! implemented by https://github.com/valknight/Cohost.py/pull/8 and release post on cohost: https://cohost.org/vallerie/post/217954-cohost-py-has-arrive

**Comment by jerwarren** [comment]
Thank you!
